submissions/TaniaBondarenko/frogger_game/app.js

```diff
@@ -0,0 +1,117 @@
+const borders = {
+    rightBorder: 400,
+    leftBorder: 5,
+    upBorder: 410,
+    downBorder: 70
+};
+
+const Enemy = function (x, y) {
+    this.sprite = 'images/enemy-bug.png';
+    this.x = x;
+    this.y = y;
+    this.speed = 75;
+
+    this.render = function () {
+        ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+    };
+};
+
+    Enemy.prototype.update = function (dt) {
+        this.x += this.speed * dt;
+        if (this.x >= 500) {
+            this.x = -10;
+            this.speed = Math.floor(40 + Math.random() * 100);
+        }
+    };
```

Comment:
we have the excess indentation here

Reply:
Please clarify

Reply:
there is extra white space before the definition of Enemy.prototype.update()

![to_left](https://user-images.githubusercontent.com/2924306/102871014-8aae2480-4446-11eb-84c3-0e348f261174.gif)

it's just about code formatting...

Reply:
Ah, okay. I've already changed all according to your comments, except this one). I'll skip it for now and will keep it in my mind for a future.